cortex/webgl/view.py

```diff
@@ -184,7 +185,8 @@ def make_static(outpath, data, types=("inflated",), recache=False, cmap="RdBu_r"
     print(disp_layers)
     loader = FallbackLoader(rootdirs)
     tpl = loader.load(templatefile)
-    kwargs.update(viewopts)
+    tpl_args = copy.deepcopy(viewopts)
+    tpl_args.update(kwargs)  # override viewopts with kwargs
```

Comment:
This has the side-effect of allowing (programmatic) users full control over the display.